CHANGELOG.md

```
@@ -0,0 +1,9 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+### [0.1.3](https://github.com/bluewave-labs/prism-design-system/compare/v0.1.2...v0.1.3) (2025-04-04)
+
+# Changelog
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)